rage/rlGamerInfoBase.hpp

```diff
@@ -11,7 +11,7 @@ namespace rage
 	class rlGamerInfoBase
 	{
 	public:
-		bool m_security_enabled;         //0x0000
+		int m_security_enabled;          //0x0000
```

Comment:
If m_peer_id starts at 0x8, shouldn't this be an int64 or at least appended by a 4-byte padding?